package.json

```diff
@@ -85,9 +85,9 @@
     "@typescript-eslint/eslint-plugin": "^6.7.0",
     "@typescript-eslint/parser": "^6.7.0",
     "@wdio/allure-reporter": "^8.20.0",
-    "@wdio/cli": "^8.20.2",
-    "@wdio/devtools-service": "^8.20.0",
-    "@wdio/local-runner": "^8.20.0",
+    "@wdio/cli": "^8.32.2",
+    "@wdio/devtools-service": "^8.32.2",
+    "@wdio/local-runner": "^8.32.2",
```

Comment:
My guess is WDIO expects all packages to be the same version. Shouldn't we also update `@wdio/allure-reporter`, `@wdio/mocha-framework` and `@wdio/spec-reporter`?

Reply:
Yea, I'm upgrading those too.